### PR TITLE
fixed genrate.ps1 and generat.cmd paths

### PR DESCRIPTION
**Pull request by chidozieononiwu**

### CODE REVIEW

**Comment by chidozieononiwu** [comment]
@Azure/azure-sdk-eng

**Comment by weshaggard** [comment]
@shahabhijeet are these scripts even used now after the automation has been setup from the specs repo?

**Review by weshaggard — APPROVED** [review]

File: sdk/hdinsight/Microsoft.Azure.HDInsight.Job/src/generate.ps1 [hunk 1]
```diff
@@ -1 +1 @@
-Start-AutoRestCodeGeneration -ResourceProvider "hdinsight/data-plane" -AutoRestVersion "latest"
\ No newline at end of file
+Start-AutoRestCodeGeneration -ResourceProvider "hdinsight/Microsoft.Azure.HDInsight.Job/src" -AutoRestVersion "latest"
```
Comment (dsgouda) [inline sdk/hdinsight/Microsoft.Azure.HDInsight.Job/src/generate.ps1]: What is the intent behind these changes? `-ResourceProvider` corresponds to the relative path to `specifications` folder in REST specs repo. The paths being introduced here don't even exist

**Review by dsgouda — CHANGES_REQUESTED** [review]

**Comment by dsgouda** [comment]
@weshaggard even if automation exists, these scripts are used if RPs wish to manually generate the SDK and open a PR
Also, while @chidozieononiwu was working on modifying the repo structure we disabled PR automation for NET SDK since this involves updating path references in our DB.

File: sdk/hdinsight/Microsoft.Azure.HDInsight.Job/src/generate.ps1 (hunk 1 above)
Comment (chidozieononiwu) [reply sdk/hdinsight/Microsoft.Azure.HDInsight.Job/src/generate.ps1]: I see what you mean, I have removed those changes from the PR. It seems the .cmd files need to be updated still?

**Review by dsgouda — APPROVED** [review]
LGTM I have assumed you have tested these scripts

**Pull request merged**